monkestation/code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -0,0 +1,53 @@
+////////////
+// SPELLS //
+////////////
+
+
+/datum/spellbook_entry/lesser_summonapes
+	name = "Lesser Summon Apes"
+	spell_type = /obj/effect/proc_holder/spell/aoe_turf/conjure/lesser_summonapes
+	category = "Defensive"
```

Comment:
Cost?
Should be at least 4

Reply:
For now I will make it cost 3 and reduce the amount of summoned mobs to 4, the mobs do have timed life. The only thing that costs 4 is curse of madness which has a massive effect on the entire round